static/src/javascripts/projects/common/modules/commercial/ad-prefs.lib.js

```diff
@@ -7,6 +7,8 @@ type AdConsent = {
     cookie: string,
 };
 
+type AdConsentState = ?boolean;
```

Comment:
What is the benefit of aliasing this maybe? It might be clearer when we run into uses of `getAdConsentState` elsewhere in the codebase to explicitly know that it returns `?boolean`, rather than needing to find the custom type. Alternatively, could we be more explicit with `boolean | null` ?

```
const readConsentCookie = (cookieName: string): boolean | null => {
    const cookieVal: ?string = getCookie(cookieName);
    if (cookieVal && cookieVal.split(',')[0] === '1') return true;
    if (cookieVal && cookieVal.split(',')[0] === '0') return false;
    return null;
};
```

Reply:
might have gone overboard with this I agree. WIll set it back to `?boolean`